synedrion/benches/bench.rs

```diff
@@ -5,13 +5,15 @@ use manul::{
     dev::{run_sync, BinaryFormat, TestSessionParams, TestSigner, TestVerifier},
     signature::Keypair,
 };
-use rand_core::OsRng;
-use synedrion::{AuxGen, AuxInfo, InteractiveSigning, KeyInit, KeyShare, TestParams};
+use primeorder::FieldBytes;
```

Comment:
We only need `primeorder` for that, and it's just a type alias, perhaps we can just define it ourselves.

Reply:
You mean like `GenericArray<u8, FieldBytesSize<C>>`? What do we gain from that? We depend on `primeorder` anyway yeah?